Neos.Flow/Classes/Security/Authentication/Token/UsernamePassword.php

```diff
@@ -13,13 +13,18 @@
 
 use Neos\Flow\Annotations as Flow;
 use Neos\Flow\Mvc\ActionRequest;
+use Neos\Flow\Security\Exception\InvalidAuthenticationStatusException;
 use Neos\Utility\ObjectAccess;
 
 /**
  * An authentication token used for simple username and password authentication.
  */
 class UsernamePassword extends AbstractToken
 {
+    private const DEFAULT_USERNAME_POST_FIELD = '__authentication.Neos.Flow.Security.Authentication.Token.UsernamePassword.username';
```

Comment:
*maybe* make that public so third parties can easily use the field path? Also see the code example here: https://github.com/neos/flow-development-collection/blob/master/Neos.Flow/Documentation/TheDefinitiveGuide/PartIII/Security.rst#simple-usernamepassword-authentication

Reply:
I don't think it should be part of the API really.. I just introduced the constants to avoid the magic strings really